interface/modules/zend_modules/config/autoload/global.php

```diff
@@ -85,6 +84,18 @@
     'service_manager' => array(
         'factories' => $factories
     ),
+    // @see https://docs.zendframework.com/tutorials/i18n/
+    // TODO: we don't seem to have any translation files here... so we may just want to remove this.
```

Comment:
openemr has its own translation mechanism.

Reply:
I doubt our module writers will be bundling their own translations for now so I'll go ahead and remove that.  I did write an adapter translator class where we do use translations in some of the modules to just call the OpenEMR translation functions.